### PR TITLE
Closes #13 - Make scripts work with Mac OS X

### LINKED ISSUE #13: Make scripts work with Mac OS X
Some commands in the `fluo-dev` script are incompatible with Mac OS X.  The script should recognize the user's OS and pick the right command to use.

### PR DESCRIPTION
**Pull request by mikewalch**
PR for #13

### CODE REVIEW

File: bin/impl/load-env.sh [hunk 1]
```diff
@@ -89,3 +89,11 @@ fi
 : ${ACCUMULO_USER:?"ACCUMULO_USER is not set in env.sh"}
 : ${ACCUMULO_PASSWORD:?"ACCUMULO_PASSWORD is not set in env.sh"}
 : ${ACCUMULO_TABLE:?"ACCUMULO_TABLE is not set in env.sh"}
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+  export MD5=md5
```
Comment (keith-turner) [inline bin/impl/load-env.sh]: Do these need to be exported since the fluo-dev script sources `load-env.sh`?

Comment (mikewalch) [inline bin/impl/load-env.sh]: The `export` is needed.  Without `export`, the vars are available to `fluo-dev` but won't be to any process started by `fluo-dev`.  Adding export makes the var available to subprocesses.  I tested it to verify.  Also, see http://stackoverflow.com/questions/1158091/defining-a-variable-with-or-without-export

Comment (keith-turner) [inline bin/impl/load-env.sh]: ok  I see.  `fluo-dev` does not source the scripts in `impl` but launches them as sup processes.

File: conf/fluo/.gitignore [hunk 2]
```diff
@@ -1,3 +1,4 @@
 fluo.properties
+fluo.properties.bak
```
Comment (keith-turner) [inline conf/fluo/.gitignore]: is this something the scripts create?

Comment (mikewalch) [inline conf/fluo/.gitignore]: yes.. on mac os x, in-place sed creates a back up with *.bak postfix

**Pull request merged**